src/components/menus/TertiaryNav/TertiaryNav.tsx

```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 import classnames from 'classnames';
-import { Switch, Route, NavLink, RouteComponentProps } from 'react-router-dom';
-import { withRouter } from 'react-router';
+import { Switch, Route, NavLink, RouteComponentProps, withRouter } from 'react-router-dom';
```

Comment:
Whoa, this actually makes a lot of sense as to why this was breaking based on those issues I was reading! Dang. Good hustle find this 👊